src-python/lock-keeper/lockkeeper.py

```diff
@@ -131,6 +132,32 @@ def fl_restart():
     return jsonify({'status': 'ok'})
 
 
+@app.route('/floodlight/tc', methods=['POST'])
+def floodlight_shape_traffic():
+    body = request.get_json()
+    common_commands = ['tc qdisc del dev eth0 root',
+                       'tc qdisc add dev eth0 root handle 1: prio priomap 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2',
+                       'tc qdisc add dev eth0 parent 1:1 handle 10: netem delay {}ms'.format(
+                           body['tcData']['egressDelayMs'])]
+    switches_by_container = defaultdict(list)
+    for sw in body['affectedAddresses']:
+        switches_by_container[sw['containerName']].append(sw)
+    for container, switches in switches_by_container.items():
+        commands = common_commands.copy()
+        for switch in switches:
+            commands.extend(['tc filter add dev eth0 protocol ip parent 1:0 prio 1 u32 match ip dst {}/32 match ip '
```

Comment:
Do we always use the `eth0` interface?
Probably would be better to use separate function for getting interface name, for example: 
`ip route show to match switch_address | awk  '{ print $5 }'`

Reply:
We always use eth0. This code is executed inside floodlight container, not lab service

Reply:
agree